sunpy/map/mapbase.py

```diff
@@ -17,6 +17,12 @@
 from matplotlib import cm
 from matplotlib.backend_bases import FigureCanvasBase
 from matplotlib.figure import Figure
+DASK_INSTALLED = False
+try:
+    from dask.array import Array as DaskArray
+    DASK_INSTALLED = True
+except ImportError:
+    pass
```

Comment:
```suggestion
try:
    from dask.array import Array as DaskArray
    DASK_INSTALLED = True
except ImportError:
    DASK_INSTALLED = False
```
I personally prefer this.

Reply:
I like this as well. Made changes as requested.